dev-workflow-ui/src/ch/ivyteam/workflowui/tasks/TasksDataModel.java

```diff
@@ -103,9 +103,6 @@ private static void applyOrdering(TaskQuery query, String sortField, SortOrder s
     if (StringUtils.isEmpty(sortField)) {
       applySorting(query.orderBy().startTimestamp(), SortOrder.DESCENDING);
     }
-    if ("state".equals(sortField)) {
-      applySorting(query.orderBy().state(), sortOrder);
```

Comment:
can't you use orderBy().businessState() ?

Reply:
... glad that I asked 🤦 
sure that fixes it 👍